### ISSUE TITLE
EventProf Minitest support

### ISSUE THREAD

**Issue by palkan** [palkan/test-prof#21]
Add Minitest integration support to EventProf.

### Tracking

I propose to use [minitest-reporters](https://github.com/kern/minitest-reporters) for tests tracking purposes (at least for now). It works pretty similar to RSpec reporter listeners, so could be easily integrated with EventProf core functionality.

### Configuration

We should support both env variables and CLI options (thus, we have to provide a Minitest plugin (see [Rails plugin](https://github.com/rails/rails/blob/master/railties/lib/minitest/rails_plugin.rb), for example)):

```
# that should work
EVENT_PROF=sql.active_record rake test

# and that too
ruby my_test.rb --event_prof=sql.active_record
```

**Comment by IDolgirev** [comment]
Hi!
Well, I believe I'm quite close to finalize this but I've faced with problem to get location of example group for proper report results.

Technically, any Minitest reporter receives such group as a Class object and example name as String in prerecord hook. I can track down location for examples via `source_location`, but can't find a robust solution to get the same for group definition.

Possible solution is to get location of the first example of the group (probably we can subtract one line and believe that there is no blank line between group definition and example definition ) and use it as a group location.
Another one is to explicitly show in group location something like 'FILE: around line XX', where XX is the location of the first group example as well.

Any thoughts or ideas?
Thanks in advance!

**Comment by palkan** [comment]
Hi!

I think, we can just use a file path without the line number for a group location. WDYT?

**Comment by IDolgirev** [comment]
I like such simplicity :)

**Comment by palkan** [comment]
Closed by #29
